WatchApp Extension/Controllers/ActionHUDController.swift

```diff
@@ -68,6 +68,23 @@ final class ActionHUDController: HUDInterfaceController {
         } else if overrideButtonGroup.state == .disabled {
             overrideButtonGroup.state = .off
         }
+        
+        requestContextUpdate()
+    }
+    
+    private func requestContextUpdate() {
+        try? WCSession.default.sendContextRequestMessage(WatchContextRequestUserInfo(), completionHandler: { (result) in
+            DispatchQueue.main.async {
+                self.pendingMessageResponses -= 1
```

Comment:
Is the corresponding `pendingMessageResponses += 1`, before the async call is made, missing here?